configure

```diff
@@ -2564,9 +2564,7 @@ case $_host_os in
 		esac
 
 		append_var DEFINES "-DMACOSX"
-		append_var LIBS "-framework AudioUnit -framework AudioToolbox -framework Carbon -framework CoreMIDI"
-		# SDL2 doesn't seem to add Cocoa for us.
-		append_var LIBS "-framework Cocoa"
+		append_var LIBS "-framework AudioUnit -framework AudioToolbox -framework Cocoa -framework CoreMIDI"
```

Comment:
SDL 1.2 needs Carbon, so we cannot just blindly remove it here. Actually SDL2 also still depends on it as well. But interestingly I do seem to get all of those when I use `sdl2-config --static-libs`, which is used in the `ports/mk` file. So I am not actually sure we need any of these here.

I would like some more time to check this.